lib/read_only/dl_esm_inf/read_only.f90

```diff
@@ -33,56 +33,29 @@
 ! -----------------------------------------------------------------------------
 ! Authors J. Henrichs, Bureau of Meteorology
 
-!> This module implements a verification that read-only fields (in the LFRic
-!! infrastructure) are not overwritten (due to memory overwrites etc).
+!> This module implements a verification that read-only fields (in the
+!! dl_esm_inf infrastructure) are not overwritten (due to memory overwrites etc).
```

Comment:
First one is done. Second one isn't since this is not a jinja created file. I've added a description of using jinja or not for the actual libraries to the documentation as well to make this difference more obvious. And I've also removed all intermediate files (i.e. jinja output files) from the repo (as agreed upon in our telco).